### PR TITLE
Fix issues with XSD not reflecting the latest ruleset

### PR DESCRIPTION
**Pull request by mbriskar**

### CODE REVIEW

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://wonka.mw.lab.eng.bos.redhat.com/jenkins//job/windup-pr-builder/1274/

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://wonka.mw.lab.eng.bos.redhat.com/jenkins//job/windup-core-PR-dependents-check/648/<h2>Failed Tests: <span class='status-failure'>1</span></h2><h3><a name='windup-core-PR-dependents-check/org.jboss.windup.rules:windup-rulesets' /><a href='http://wonka.mw.lab.eng.bos.redhat.com/jenkins/job/windup-core-PR-dependents-check/648/org.jboss.windup.rules$windup-rulesets/testReport'>windup-core-PR-dependents-check/org.jboss.windup.rules:windup-rulesets</a>: <span class='status-failure'>1</span></h3><ul><li><a href='http://wonka.mw.lab.eng.bos.redhat.com/jenkins/job/windup-core-PR-dependents-check/648/org.jboss.windup.rules$windup-rulesets/testReport/org.jboss.windup.rules.tests/WindupRulesTest/testWindupRules/'><strong>org.jboss.windup.rules.tests.WindupRulesTest.testWindupRules</strong></a></li></ul>

File: rules-base/api/src/main/java/org/jboss/windup/rules/files/condition/ToFileModelHandler.java [hunk 1]
```diff
@@ -35,22 +35,14 @@
...
-        Object wrappedCondition = context.processElement(firstChild);
-        validateWrappedCondition(wrappedCondition);
+        GraphCondition wrappedCondition = (GraphCondition) context.processElement(firstChild);
 
-        // Create the condition
-        ToFileModel toFileModelCondition = new ToFileModel();
-        for (Element child : children)
-        {
-            Object condition = context.processElement(child);
-            toFileModelCondition.withWrappedCondition((GraphCondition) condition);
```
Comment (mbriskar) [inline rules-base/api/src/main/java/org/jboss/windup/rules/files/condition/ToFileModelHandler.java]: this was a bug found. withWrappedCondition returns a new Condition and does not change the current one (the method is static)

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://wonka.mw.lab.eng.bos.redhat.com/jenkins//job/windup-pr-builder/1276/

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://wonka.mw.lab.eng.bos.redhat.com/jenkins//job/windup-core-PR-dependents-check/650/

**Pull request merged**